frontend/src/components/StaticTokenKubeconfigConfiguration.vue

```diff
@@ -0,0 +1,74 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <action-button-dialog
+    :shoot-item="shootItem"
+    @dialog-opened="onConfigurationDialogOpened"
+    ref="actionDialog"
+    width="450"
+    confirm-required
+    caption="Configure Admin Kubeconfig">
+    <template v-slot:actionComponent>
+      <static-token-kubeconfig-switch
+        v-model="enableStaticTokenKubeconfig"
+      ></static-token-kubeconfig-switch>
+    </template>
+  </action-button-dialog>
+</template>
+
+<script>
+import ActionButtonDialog from '@/components/dialogs/ActionButtonDialog'
+import StaticTokenKubeconfigSwitch from '@/components/StaticTokenKubeconfigSwitch'
+import { updateShootEnableStaticTokenKubeconfig } from '@/utils/api'
+import { errorDetailsFromError } from '@/utils/error'
+import shootItem from '@/mixins/shootItem'
+
+export default {
+  name: 'StaticTokenKubeconfigConfiguration',
```

Comment:
```suggestion
  name: 'static-token-kubeconfig-configuration',
```

Reply:
>🤔 The idea was to show the divider if the previous tile is visible and any of the following tiles is visible. Why is the check for the following tiles not necessary anymore?

Why would we need it if kubeconfig tile is always visible